Azure/azure-cli
# {Core} Copy login credential when run test with `--live` and `random_config_dir` is enabled

**Description**

As mentioned in https://github.com/Azure/azure-cli/pull/26273#issuecomment-1551174298, the login info can't be retrieved in recording mode when enable `random_config_dir`,
This PR copies the login credential from `~/.azure` to random config dir.

Related PR: https://github.com/Azure/azure-cli/pull/25689

**Testing Guide**

`azdev test test_tag_update_by_patch --live`
Before:
`FAILED src\azure-cli\azure\cli\command_modules\resource\tests\latest\test_resource.py::TagScenarioTest::test_tag_update_by_patch - knack.util.CLIError: Please run 'az login' to setup account.`
After:
`Message: The client 'xxx' with object id 'xxx' does not have authorization to perform action 'Microsoft.Resources/subscriptions/resourcegroups/write' over scope '/subscriptions/axx' or the scope is invalid. If access was recently granted, please refresh your credentials.`
So it can read the token.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
